src/client.ts

```diff
@@ -871,4 +883,115 @@ export class C8Client extends Fabric {
+  }
+
+  deletePlan(planName: string) {
+    const plan = this.plan(planName);
+    return plan.deletePlan();
+  }
+
+  getPlanDetails(planName: string) {
+    const plan = this.plan(planName);
+    return plan.getPlanDetails();
+  }
+
+  updatePlan(planName: string, planDetails: PlanDetails) {
+    const plan = this.plan(planName);
+    return plan.updatePlan(planDetails);
+  }
+
+  updateTenantPlan(updateTenantPlan: UpdateTenantPlan) {
+    const plan = this.plan();
+    return plan.updateTenantPlan(updateTenantPlan);
+  }
+
+  /**plans apis ends here */
+
+  /** billing apis starts from here */
+
+  billing(tenantName: string) {
+    return new Billing(this._connection, tenantName);
+  }
+
+  getAccountDetails(tenantName: string) {
+    const billing = this.billing(tenantName);
+    return billing.getAccountDetails();
+  }
+
+  updateAccountDetails(tenantName: string, accountDetails: AccountDetails) {
+    const billing = this.billing(tenantName);
+    return billing.updateAccountDetails(accountDetails);
+  }
+
+  updatePaymentSettings(tenantName: string, paymentMethodId: string) {
+    const billing = this.billing(tenantName);
+    return billing.updatePaymentSettings(paymentMethodId);
+  }
+
+  getPaymentDetailsOfPreviousMonths(tenantName: string, limit: number) {
+    const billing = this.billing(tenantName);
+    return billing.getPaymentDetailsOfPreviousMonths(limit);
+  }
+
+  getInvoices(tenantName: string, limit: number) {
+    const billing = this.billing(tenantName);
+    return billing.getInvoices(limit);
+  }
+
+  getCurrentInvoices(tenantName: string) {
+    const billing = this.billing(tenantName);
+    return billing.getCurrentInvoices();
+  }
+
+  getInvoiceOfSpecificMonthYear(tenantName: string, year: number, month: number) {
+    const billing = this.billing(tenantName);
+    return billing.getInvoiceOfSpecificMonthYear(year, month);
+  }
+
+  getUsageOfTenant(tenantName: string, startDate?: string, endDate?: string) {
+    const billing = this.billing(tenantName);
+    return billing.getUsageOfTenant(startDate, endDate);
+  }
+
+  getUsageOfTenantForSpecificRegion(tenantName: string, region: string, startDate?: string, endDate?: string) {
+    const billing = this.billing(tenantName);
+    return billing.getUsageOfTenantForSpecificRegion(region, startDate, endDate);
+  }
+
+  /** billing apis ends here */
+
+
+  /**export and import apis starts from here */
+
+  importAndExport(collectionName: string = "") {
+    return new ImportAndExport(this._connection, collectionName)
+  }
+
+  getDataByQuery(query: string) {
+    return this.importAndExport().getDataByQuery(query)
+  }
+
+  getDataByCollectionName(collectionName: string, params: CollectionParams = {}) {
+    return this.importAndExport(collectionName).getDataByCollectionName(params)
+  }
+
+  createDocuments(collectionName: string, data: string[], showErrors: boolean) {
+    return this.importAndExport(collectionName).createDocuments(data, showErrors)
```

Comment:
- this doesn't look clean. keep the way we are using. like add instance on one line and api use in other line.
- add semicolons for all statements.

Reply:
updated